coqide-server.opam

```diff
@@ -1,15 +1,26 @@
-opam-version: "1.2"
+synopsis: "The Coq Proof Assistant"
+description: """
+Coq is a formal proof management system. It provides
+a formal language to write mathematical definitions, executable
+algorithms and theorems together with an environment for
+semi-interactive development of machine-checked proofs.
+
+This package provides the `coqidetop` language server, an
+implementation of Coq's (XML
+protocol)[https://github.com/coq/coq/blob/master/dev/doc/xml-protocol.md]
```

Comment:
If you meant to use Markdown's standard for link you have it backwards (should be `[ ]( )`). But what about not using Markdown? I'm not sure it's supported here.